### PR TITLE
Adds grantee certification process

### PR DESCRIPTION
**Pull request by antonykhaemba**
This is the Accelerator program certification guidelines

### CODE REVIEW

File: CERTIFICATION.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+### 2. Certification of grantee organizations.
+The Accelerator organizations will be certified if they demonstrate that after being onboarded on CHT, the organizations are able to design and build CHT workflows and use cases
+
+#### Evaluation criteria 
+The design process and outputs should be well documented
+- The user requirements, technical design guides and prototypes should be well documented.
+  
+**Recommended configuration best practices and configuration outputs**
+- The code should follow the recommended [coding style](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/).
```
Comment (esthermmoturi) [inline CERTIFICATION.md]:
```suggestion
- The code should follow the recommended [coding style](https://github.com/medic/cht-accelerator#readme).
```

Comment (esthermmoturi) [reply CERTIFICATION.md]: @antonykhaemba Can we use the readme file instead?It is more detailed.

File: CERTIFICATION.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+## Accelerator Program Certification Process.
+The following guidelines will be used in certifying the CHT Accelerator program grantees. App Developers and grantee organizations will be certified after completing the CHT app building course and also demonstrating that after the CHT onboarding the organizations are able to design and build CHT prototypes or workflows.
```
Comment (esthermmoturi) [inline CERTIFICATION.md]:
```suggestion
The following guidelines will be used in certifying the CHT Accelerator program grantees. App Developers and grantee organizations will be certified after completing the CHT app building course and demonstrating that after the CHT onboarding process, the organizations are able to design and build CHT prototypes or workflows. 
```

**Review by esthermmoturi — CHANGES_REQUESTED** [review]

File: CERTIFICATION.md (hunk 1 above)
Comment (antonykhaemba) [reply CERTIFICATION.md]: Yes we can use that.

**Review by esthermmoturi — APPROVED** [review]
@antonykhaemba LGTM!

**Pull request merged**